src/components/seeds/lab/index.tsx

```diff
@@ -187,9 +187,9 @@ export default function Nursery({ accession, onSubmit, organization }: Props): J
 
   const createCutTestRow = () => {
     const cutTest = {
-      compromisedSeeds: accession.cutTestSeedsCompromised ?? '--',
-      emptySeeds: accession.cutTestSeedsEmpty ?? '--',
-      filledSeeds: accession.cutTestSeedsFilled ?? '--',
+      compromisedSeeds: accession.viabilityTests_seedsCompromised ?? '--',
+      emptySeeds: accession.viabilityTests_seedsEmpty ?? '--',
+      filledSeeds: accession.viabilityTests_seedsFilled ?? '--',
```

Comment:
Also unsure if the changes in this file are correct or needed?

Reply:
We don't use this lab file anymore. So this is not needed but if it throw error, then you can leave the changes

Reply:
I will be removing this code as part of https://terraformation.atlassian.net/browse/SW-1449 - after you are done.

Reply:
Ok got it, I left the changes since the file is being removed anyways.